includes/classes/blocks/class-query-loop.php

```
@@ -518,7 +519,13 @@ public function related_connection_query( $items, $to, $from ) {
 			return $items;
 		}
 
-		$found_items = get_post_meta( get_the_ID(), $to . '_to_' . $from, true );
+		/**
+		 * lsx_to_related_connection_found_items filter.
+		 * @var $key
+		 * @var $current_item_ID
+		 */
+		$key         = apply_filters( 'lsx_to_related_connection_query_key', $to . '_to_' . $from, get_the_ID() );
+		$found_items = get_post_meta( get_the_ID(), $key, true );
```

PHPMD (2.15.0)

The variable $found_items is not named in camelCase. (undefined)

(CamelCaseVariableName)